react/Modal/styles.styl

```diff
@@ -59,7 +59,7 @@
 .c-app-editor
```

Comment:
Maybe we could also change here for more consistency?

Reply:
Yes, but it's not in the scope of this PR and should be done in a cleaning/refacto :)